### ISSUE TITLE
[BUG] VST3 not showing up in Ableton Live on MacOS

### ISSUE THREAD

**Issue by jonahclarsen** [magenta/ddsp-vst#34]
**Describe the bug**
After installing the .vst3 files to the appropriate directory, the effect doesn't appear in Ableton Live. The AU version works fine.

**System Info**
- M2 Macbook Air, MacOS 13.1
- Ableton Live 11.2.7
- v1.1.0

**Comment by ypj95** [comment]
Hi,

I might have a solution for you: You'll find the Plug-Ins when running Ableton through Rosetta 2.


I guess the plug-in is conflicting with the m1/m2 chip architecture? 

Tested on:

- m1 MacBook pro macOS 13.1
- Ableton Live 11.2.5

**Comment by jonahclarsen** [comment]
> Hi,
> 
> I might have a solution for you: You'll find the Plug-Ins when running Ableton through Rosetta 2.
> 
> I guess the plug-in is conflicting with the m1/m2 chip architecture?
> 
> Tested on:
> 
> * m1 MacBook pro macOS 13.1
> * Ableton Live 11.2.5

Ah, thank you for that - it's likely that the VST has only been compiled for Intel CPUs and not for Apple Silicon ones. Unfortunately I can't use Rosetta 2 because it makes my projects much slower. Thanks anyway though, hopefully the devs can compile it for both.

**Comment by tekkno-primitiv** [comment]
++! Please, compile vst3 for apple architecture

**Comment by talaviram** [comment]
Indeed, the provided binaries of the project are compiled as x86_64 only:

``
file ~/Library/Audio/Plug-Ins/VST3/DDSP\ Effect.vst3/Contents/MacOS/DDSP\ Effect 
/Users/user/Library/Audio/Plug-Ins/VST3/DDSP Effect.vst3/Contents/MacOS/DDSP Effect: Mach-O 64-bit bundle x86_64
``
